system-tests/test/testConfigOverrides_spec.ts

```diff
@@ -33,6 +33,24 @@ describe('testConfigOverrides', () => {
     },
   })
 
+  systemTests.it(`correctly applies overrides when valid configuration for describe.only`, {
+    spec: 'testConfigOverrides-describe-only-valid.js',
+    snapshot: true,
+    expectedExitCode: 0,
+    config: {
+      video: false,
+    },
+  })
+
+  systemTests.it(`correctly applies overrides when valid configuration for it.only`, {
+    spec: 'testConfigOverrides-it-only-valid.js',
+    snapshot: true,
+    expectedExitCode: 0,
+    config: {
+      video: false,
+    },
+  })
```

Comment:
I think these don't need to be system tests, they can be regular driver Cypress tests instead, right? Since we run one test file at a time in `cypress run`, it should work the same. `packages/driver/cypress/integration/e2e` would be a good place.

Avoid adding system tests if possible since they're a lot heavier, resource-wise. And since we're just running the test and the snapshot doesn't have any additional info in it, it should be fine to make these regular Cypress tests.

Reply:
hmmm good point. Updated: [`96e4e80` (#18961)](https://github.com/cypress-io/cypress/pull/18961/commits/96e4e80b4fe75f10da49b386c0ec33b01a8f0a02)